scripts/i18n-doc-scan.sh

```
@@ -0,0 +1,92 @@
+#!/bin/bash
+# i18n-doc-scan.sh - OpenHuman GitBook 中文文档问题扫描
+
+set -e
+
+echo "=== OpenHuman i18n 文档扫描 ==="
+echo ""
+
+# 1. 未本地化的内部链接
+echo "【1/5】检查未本地化的 .md 链接..."
+UNLOCALIZED=$(find gitbooks -name "*.zh-CN.md" -exec grep -l '\.md)' {} \; 2>/dev/null | while read f; do
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)